From ehovland@s383.jpl.nasa.gov Thu Oct  6 10:49:42 2005
Date: Thu, 6 Oct 2005 10:47:49 -0700
From: Erik Hovland <erik@hovland.org>
To: gregkh@suse.de
Cc: trivial@rustcorp.com.au
Subject: [trivial patch] changes device to driver in porting.txt
Message-ID: <20051006174749.GH25998@mage.jpl.nasa.gov>

The document porting.txt in Documentation/driver-model says:
When a device is successfully bound to a device

I think it should say:
When a device is successfully bound to a driver


Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 Documentation/driver-model/porting.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/Documentation/driver-model/porting.txt
+++ gregkh-2.6/Documentation/driver-model/porting.txt
@@ -350,7 +350,7 @@ When a driver is registered, the bus's l
 over. bus->match() is called for each device that is not already
 claimed by a driver. 
 
-When a device is successfully bound to a device, device->driver is
+When a device is successfully bound to a driver, device->driver is
 set, the device is added to a per-driver list of devices, and a
 symlink is created in the driver's sysfs directory that points to the
 device's physical directory:
